decidim-core/lib/decidim/authorable.rb

```diff
@@ -39,6 +39,20 @@ def official?
         decidim_author_type == Decidim::Organization.name
       end
 
+      # Public: Checks whether the resource is created by a or not.
+      #
+      # Returns a boolean.
+      def group?
+        decidim_user_group_id.present?
```

Comment:
I'd rather check the presence of `user_group`, to make sure the group exists and not just the ID.